rp2040-hal/src/pio.rs

```diff
@@ -1225,7 +1328,6 @@ unsafe impl<SM: ValidStateMachine + Send> Send for Tx<SM> {}
 // are added.
 impl<SM: ValidStateMachine> Tx<SM> {
     fn register_block(&self) -> &pac::pio0::RegisterBlock {
-        // Safety: The register is unique to this Tx instance.
```

Comment:
Just removing the comment doesn't make the code correct :-)
(But I'll have a closer look if there's really something unsafe here, and if necessary, open a ticket about it. That's not related to this PR)